File: programmers/난이도별/level01.두_개_뽑아서_더하기/devSOWON0628.js
function solution(numbers) {
    var answer = [];
    for(var i =0 ;i < numbers.length ; i++){
        for (var j= 0 ; j< numbers.length ; j++){
            if(i==j)continue;
            answer.push(numbers[i] + numbers[j]);
        }
    }
    const set = Array.from(new Set(answer)); 
    return set.sort((a, b) => a - b)
}

File: programmers/난이도별/level01.음양_더하기/devSOWON0628.java
class Solution {
    public int solution(int[] absolutes, boolean[] signs) {
        int answer = 0;
        for(int i = 0; i < absolutes.length ;i++){
            if(signs[i]){
                answer+=absolutes[i];
            }else{
                answer-=absolutes[i];
            }
        }
        return answer;
    }
}
